### DISCUSSION TITLE
Update row with importFromJson()

### DISCUSSION THREAD

**Discussion by aanders77** [capacitor-community/sqlite#355] [Q&A] [answered]
Hi! I was wondering if it is possible to update only some fields of a table row with importFromJson(), but leave the other fields unchanged?

Example: Let's say I have a table with columns `ID [Primary key], Name, Age`. In that table I have a row with values `[40, 'Peter', 27]`.
Would it be possible to update Peter's age using importFromJson(), but at the same time leave the name column unchanged?

**Comment by jepiqueau** [comment, marked as answer]
@aanders77 thanks for your question. The answer is no. ImportFromJson import row by row in a partial mode for each row which have been modified on the remote database according to the last_modified date.

**Reply by aanders77** [reply]
Ok, thanks for the reply. I found a different way to solve my problem 👍

**Comment by jepiqueau** [comment]
@aanders77 OK so i close the issue. Thanks for using the plugin
